lib/npm/tagger.js

```diff
@@ -29,27 +29,48 @@ var Tagger = module.exports = function Tagger(opts) {
  * @returns {undefined}
  */
 Tagger.prototype.delete = function (opts, callback) {
-  var pkg = opts.pkg,
-    tag = opts.tag;
-
-  var update = {
-    name: pkg,
-    distTags: {}
-  };
-
-  //
-  // Note that for maps we actually don't currently delete values as to
-  // prevent tombstones. This should technically be allowed but we should
-  // expect in this case for the value to still exist but have a null value
-  //
-  update.distTags[tag] = null;
+  this._createBody(opts, (error, { update } = {}) => {
+    if (error) return callback(error);
+
+    //
+    // Note that for maps we actually don't currently delete values as to
+    // prevent tombstones. This should technically be allowed but we should
+    // expect in this case for the value to still exist but have a null value
+    //
```

Comment:
this was an old Cassandra comment which can be removed.

Reply:
Should the `null` become a `delete` then?

Reply:
nvm just seen your comment, thanks